### ISSUE TITLE
Add an ST section

### ISSUE THREAD

**Issue by kerrtravers** [scratchuserstats/scratchstats#21]
Would it be possible to add a section stating whether or not that user is part of the Scratch Team?

**Comment by Sheshank-s** [comment]
Hmm interesting. I think rather than using the api for this we could have an array of all the ST members @WorldLanguages

**Comment by kerrtravers** [comment]
That does seem the best option, we could get everything from the credits page and the members of the organization.

**Comment by WorldLanguages** [comment]
The Scratch API does not include if the user is a Scratcher, New Scratcher or Scratch Team member, so closing.

**Comment by joker314** [comment]
You could scrape?

**Comment by Sheshank-s** [comment]
@joker314 i thought about that..

**Comment by joker314** [comment]
Yeah, scraping the profile page and using a dirty Regular Expression to select the Scratcher status?

**Comment by kerrtravers** [comment]
@joker314 @GreenBayRules All the ST members are on https://scratch.mit.edu/info/credits. Does that help?

**Comment by Sheshank-s** [comment]
I know, @ktrvs 

I think I might be able to do this, but my only concern is it will slow down the site.

**Comment by joker314** [comment]
@GreenBayRules The site already makes tons of API calls anyway -- that's the point of it 😛 -- and they're asynchronous anyway (you can always load it last if you're concerned). 🙂

**Comment by Sheshank-s** [comment]
@joker314 I know, but I'm thinking maybe having an array of users that are in the Scratch Team might be more efficient.

**Comment by joker314** [comment]
1) The whole point is that you use API requests to fetch data (why not keep an array of projects on the "explore" page rather than having it fetch them?)

2) ST members come and go

🙂

**Comment by Sheshank-s** [comment]
Good point I guess, so okay sure. I'll get this done on monday since i need to go some where else over the weekend.

**Comment by Waakul** [comment]
> The Scratch API does not include if the user is a Scratcher, New Scratcher or Scratch Team member, so closing.

tf, it does!
EDIT: realized that it wasnt in the old scratch api (i.e: scratch.mit.edu/api/v1)
